qark/modules/DetermineMinSDK.py

```diff
@@ -75,6 +75,9 @@ def determine_min_sdk():
                 determineSdk = 'm'
             else:
                 logger.info(common.config.get('qarkhelper', 'MIN_SDK_VERSION') + str(common.minSdkVersion))
+        elif not common.interactive_mode:   # Need to set a default if in automated mode, cannot ask the user
+            common.minSdkVersion = 7
+            return
```

Comment:
When running in non-interactive mode, the user is not available to be bothered if there is wasn't a minimum SDK version found in the manifest. I chose 7 as the default, because that was the default for "manual" option if the user said "no" to entering a version.